pkg/common-controller/snapshot_controller_base.go

```diff
@@ -20,10 +20,10 @@ import (
 	"fmt"
 	"time"
 
-	crdv1 "github.com/kubernetes-csi/external-snapshotter/v2/pkg/apis/volumesnapshot/v1beta1"
-	clientset "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/clientset/versioned"
-	storageinformers "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/informers/externalversions/volumesnapshot/v1beta1"
-	storagelisters "github.com/kubernetes-csi/external-snapshotter/v2/pkg/client/listers/volumesnapshot/v1beta1"
+	crdv1 "github.com/kubernetes-csi/external-snapshotter/client/v2/apis/volumesnapshot/v1beta1"
```

Comment:
I think the plan is to remove v2 from the physical path and only change the package name to include v2 in go.mod.  Is it not possible?

Reply:
Hi @xing-yang I have tried it but if i didn't add v2 in import path then this throw error for not finding the package that include v2. 
For example in  the go.mod file for client to include v2 
crdv1 "github.com/kubernetes-csi/external-snapshotter/client/apis/volumesnapshot/v1beta1"
The imports without "v2" is failing,So i added v2 in physical path and remove V2 folder.

Reply:
Can you take a look to see how kubernetes/kubernetes is using klog v2?

See the changes in go.mod to import log v2:
https://github.com/kubernetes/kubernetes/blob/master/go.mod#L128
https://github.com/kubernetes/kubernetes/blob/master/go.mod#L477

Reply:
Our sidecars including external-snapshotter has not been updated to use klog yet.  So k/k is a better example.

Reply:
@xing-yang sure i will take a look on this

Reply:
@xing-yang in this commit , i have removed v2 folder and v2 now only used in module path.Could you clarify on what changes you are mentioning in the [comment](https://github.com/kubernetes-csi/external-snapshotter/pull/307#discussion_r449126050) .